workflow/setup_expt.py

```diff
@@ -108,47 +111,51 @@ def fill_EXPDIR(inputs):
     return
 
 
+def update_configs(host, inputs):
+
+    # First update config.base
+    edit_baseconfig(host, inputs)
+
+    yaml_path = inputs.yaml
```

Comment:
Instead of looking for a specific filename, can the filename be an optional keyword argument to setup_expt.py?

Reply:
Duh, there's not quotation marks making this a string. And I somehow missed the new argument, despite looking for it. My brain must still be on leave.